pkg/apis/serving/v1/configuration_defaults_test.go

```diff
@@ -156,6 +157,59 @@ func TestConfigurationDefaulting(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "run latest with identical timeout defaults",
+		in: &Configuration{
+			Spec: ConfigurationSpec{
+				Template: RevisionTemplateSpec{
+					Spec: RevisionSpec{
+						PodSpec: corev1.PodSpec{
+							EnableServiceLinks: ptr.Bool(true),
+							Containers: []corev1.Container{{
+								Image: "busybox",
+							}},
+						},
+						ContainerConcurrency: ptr.Int64(config.DefaultContainerConcurrency),
+					},
+				},
+			},
+		},
+		want: &Configuration{
+			Spec: ConfigurationSpec{
+				Template: RevisionTemplateSpec{
+					Spec: RevisionSpec{
+						PodSpec: corev1.PodSpec{
+							EnableServiceLinks: ptr.Bool(true),
+							Containers: []corev1.Container{{
+								Name:           config.DefaultUserContainerName,
+								Image:          "busybox",
+								Resources:      defaultResources,
+								ReadinessProbe: defaultProbe,
+							}},
+						},
+						TimeoutSeconds:       ptr.Int64(423),
+						ContainerConcurrency: ptr.Int64(config.DefaultContainerConcurrency),
+					},
+				},
+			},
+		},
+		ctx: func() context.Context {
```

Comment:
The context setup logic is duplicated between configuration_defaults_test.go and
revision_defaults_test.go. Consider extracting this into a test helper function to improve
maintainability.

Reply:
Actually they are using a different config store although the func is similar.